cfme/tests/networks/test_sdn_crud.py

```diff
@@ -41,3 +43,21 @@ def test_sdn_crud(provider, appliance):
     provider.wait_for_delete()
 
     assert not network_provider.exists
+
+
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.10',
+                         reason='Delete button was added in CFME 5.10.')
+@pytest.mark.provider([OpenStackProvider], scope='function', override=True)
+def test_router_crud(provider, appliance):
+    router_name = 'test_router_crud-{}'.format(fauxfactory.gen_alphanumeric().lower())
+    routers = appliance.collections.network_routers
+    router = routers.create(router_name, provider, provider.data.networks.cloud_tenant,
```

Comment:
**Optional** Please pass the keys along with the values in this create call and break on separate lines for each key/value.